src/wikitextprocessor/luaexec.py

```diff
@@ -84,33 +84,19 @@ def lua_loader(ctx: "Wtp", modname: str) -> Optional[str]:
     return data
 
 
+html_entities_re = re.compile(r"&(lt|gt|amp|quot|nbsp|#[xX]?[0-9A-Fa-f]+);")
```

Comment:
`A-Fa-f` must be a typo, I fixed the pattern in #247.

I guess the pr is for the Lua error in page https://en.wiktionary.org/wiki/galar, wikitext: `{{inh|ga|ine-pro|[[*ǵʰelH-]]ro-}}`

lua error: 

```
[galar (Scottish Gaelic top-level)](https://kaikki.org/dictionary/All%20languages%20combined/meaning/g/ga/galar.html) LUA error in #invoke('etymology/templates/inherited', 'inherited') parent ('Template:inherited', {1: 'gd', 2: 'ine-pro', 3: '[[*ǵʰelH-]]ro-'})

[string "Module:links"]:180: The specified language Proto-Indo-European is unattested, while the given word is not marked with '*' to indicate that it is reconstructed.

Path: galar > inh > #invoke > #invoke
```

but I couldn't find the relation between this pr and the lua error.